Domain/sdLitica.Entities/Analytics/UserAnalyticsOperation.cs

```diff
@@ -25,23 +27,23 @@ public UserAnalyticsOperation()
         }
 
 
-
-        public Guid Id { get; set; } // temporary
-
         /// <summary>
         /// Name of operation to be performed. 
         /// </summary>
+        [Column("OPERATION_NAME")]
         public string OpName { get; set; }
 
         /// <summary>
         /// Id of time-series on which operation is performed
         /// </summary>
+        [Column("TIMESERIES_INFLUX_ID")]
```

Comment:
Here, the same as above. Same names when they refer to the same thing can easy future implementations.

Reply:
will rename property of entity class

Reply:
Do we need to keep influx in the class name? What if we will change storge in the future? Maybe we can go with something like TIMESERIES_EXTERNAL_ID? And it will point to TS data in some external system, in our case Influx. Does it makes sense?

Reply:
good point. will change "influx" to "external"